code/game/objects/effects/landmarks/marine_spawns.dm

```diff
@@ -7,7 +7,6 @@
 	if(!GLOB.marine_spawns_by_job[job])
 		GLOB.marine_spawns_by_job[job] = list()
 	GLOB.marine_spawns_by_job[job] += loc
-	return INITIALIZE_HINT_QDEL

```

Comment:
Are you aware of the potential issues with this change?

Reply:
I am this leaves the `obj/effect` on the map. This is needed to ensure the spawns can move with the ship.